tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
+envlist =
+    py27, pypy, py34, py35, py36
+
+[testenv]
+commands =
+    zope-testrunner --test-path=src []
+deps =
+    git+https://github.com/zopefoundation/zope.app.rotterdam@python3#egg=zope.app.rotterdam
```

Comment:
It's been merged, so I think we're just awaiting a release. And I think the feedback has been addressed on this so it too could be merged and released.